### ISSUE TITLE
Something has been deprecated.

### ISSUE THREAD

**Issue by MakerButt** [EagleW/PaperRobot#18]
When I got into the Quickstart step, 
I typed in 
        python train.py
and everything went normal at the beginning, but later the terminal showed that something has been deprecated. And then it stopped.

python train.py
Found 23894 unique words (9146765 in total)
finish_dump
Finish loading train
Finish loading valid
Finish loading test
Epoch 0
/home/letsuya/miniconda3/envs/3.6env/lib/python3.7/site-packages/torch/nn/functional.py:1386: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
已砍掉

What should I do to fix it?
Thank you guys~

**Comment by EagleW** [comment]
@MakerButt You need to follow the environment setting:
Python3.6
Pytorch 1.1
NumPy 1.16.3
SciPy 1.2.1
NetworkX 2.3
Thanks!
